src/components/organisms/Dashboard/Tableview/Drawer.tsx

```diff
@@ -86,12 +105,13 @@ export const Drawer = () => {
           ]}
         />
         <S.TabsFooter>
-          <S.NavigationButton>
-            <S.LeftOutlined />
-          </S.NavigationButton>
-          <S.NavigationButton>
-            <S.RightOutlined />
-          </S.NavigationButton>
+          <S.ActionButtons>
+            {activeTab === 'Manifest' && (
+              <S.ActionButton disabled={!localResource} onClick={() => handleApplyResource()}>
+                Deploy
```

Comment:
Related to this https://github.com/kubeshop/monokle/issues/2753 , should we replace `Deploy` with `Update` ?